src/crypto/Ed25519PrivateKey.ts

```diff
@@ -2,10 +2,37 @@ import * as nacl from "tweetnacl";
 import * as crypto from "crypto";
 import { Ed25519PublicKey } from "./Ed25519PublicKey";
 import { Mnemonic } from "./Mnemonic";
-import { decodeHex, deriveChildKey, ed25519PrivKeyPrefix, encodeHex, pbkdf2, randomBytes } from "./util";
+import { decodeHex, deriveChildKey, ed25519PrivKeyPrefix, encodeHex, pbkdf2, randomBytes, findSubarray, arraysEqual } from "./util";
 import { RawKeyPair } from "./RawKeyPair";
 import { createKeystore, loadKeystore } from "./Keystore";
 import { BadKeyError } from "../errors/BadKeyError";
+import { BadPemFileError } from "../errors/BadPemFileError";
+
+const privateKeyUint8Array: Uint8Array =
```

Comment:
Make sure this is "-----BEGIN PRIVATE KEY-----\n"
Also what about using "\n-----END PRIVATE KEY-----\n"